docs/releases/1.11.txt

```diff
@@ -329,6 +329,9 @@ Models
 * You can now use the ``unique=True`` option with
   :class:`~django.db.models.FileField`.
 
+* The ``has_auto_field`` attribute of ``Model._meta`` is deprecated, in favour
```

Comment:
This goes in the "Features deprecated" section of the release notes rather than in "Minor features."
Suggested wording: `Model._meta.has_auto_field` is deprecated in favor of checking `if Model._meta.has_auto_field.auto_field is not None`. (use the same message in the deprecation warning).